pipeline-ui/values.yaml

```diff
@@ -17,21 +17,31 @@ service:
   port: 80
   
 ingress:
+  # -- Enable [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/).
   enabled: false
-  ingressClassName: traefik
+
+  # -- Ingress [class name](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class).
+  className: ""
+
+  # -- Annotations to be added to the ingress.
   annotations: {}
-    #kubernetes.io/ingress.class: traefik
-    #ingress.kubernetes.io/ssl-redirect: "false"
-    #traefik.frontend.rule.type: PathPrefix
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+
+  # -- Ingress host configuration.
+  # @default -- See [values.yaml](values.yaml).
   hosts:
-    # - "/"
-    # - "domain.com/xyz"
-    # - "domain.com"
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
```

Comment:
**Question**: is this non-commented part backward-compatible in resulting behavior?

Reply:
Now that I think about it, probably not because we expect an array of maps now instead of array of strings... Do you have suggestions how could this be made backward compatible? Or can we leave it this way with a warning somewhere?

Reply:
We can do it this way, but we should do a major bump IMO.